web_src/css/modules/navbar.css

```diff
@@ -133,8 +133,8 @@
   left: 6px;
   top: -9px;
   min-width: 17px;
-  min-height: 17px;
-  border-radius: var(--border-radius-circle);
+  height: 17px;
+  border-radius: 11px; /* (height + 2 * borderThickness) / 2 */
```

Comment:
Not sure about the comment but value is right, matching `height.`

Reply:
It is right.

Reply:
Ah I missed that you changed value as well. Well let's hope it works :)